rules/library.bzl

```diff
@@ -442,6 +489,33 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             tags = _MANUAL,
         )
         vendored_deps.append(import_name)
+        import_headers = native.glob(
+            ["%s/**/*.h" % vendored_dynamic_framework],
+            exclude = ["**/.DS_Store"],
```

Comment:
Nit: is it necessary to exclude .DS_Store files?

Reply:
The existing code does this so keeping that pattern up

Reply:
Edit: good catch - the glob is terminated with `.h` - so we can remove the exclusion clause 👍